python/sglang/srt/layers/attention/flashattention_backend.py

```diff
@@ -100,6 +203,14 @@ def __init__(
         self.step_id = step_id
         self.speculative_num_steps = speculative_num_steps
 
+        # Local attention settings
+        self.attention_chunk_size = (
+            model_runner.attention_chunk_size
+            if hasattr(model_runner, "attention_chunk_size")
```

Comment:
👀 is attention_chunk_size passed in somewhere in the model runner? couldn't find it currently. Thanks!

Reply:
just updated it in model_runner